Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs

```diff
@@ -140,6 +144,18 @@ private void AddFakeStore(IServiceCollection services)
     private static void AddFakeViewCompilerProvider(IServiceCollection services) =>
         services.AddSingleton<IViewCompilerProvider, FakeViewCompilerProvider>();
 
+    // We remove the existing IRecipeHarvester implementations and add a custom implementation that uses the same code
+    // as OC but with a fix in RecipeHarvester.HarvestRecipesAsync to avoid sync over async issue.
+    // This can be removed if the related issue in OC gets fixed and merged.
+    // OC issue: https://github.com/OrchardCMS/OrchardCore/issues/10329.
+    private static void AddFakeRecipeHarvester(IServiceCollection services)
+    {
+        services.RemoveAll<IRecipeHarvester>();
```

Comment:
Shouldn't you specifically remove `RecipeHarvester` and `ApplicationRecipeHarvester` instead of all implementations? We don't know if others may add their own recipe harvesters, but if they do this will break. (we add our own template harvester in Lombiq.VueJs so that feels similar)